### DISCUSSION TITLE
Rpi Zero 2w HDMI output

### DISCUSSION THREAD

**Discussion by rajattrehan21** [mikebrady/shairport-sync#1559] [Q&A]
Hi, 

Is there a way i could use Rpi zero 2w inbuilt HDMI out for audio. 

Though it works for a bit then gives error - _"fatal error: alsa: error -19 ("No such device") opening alsa device "default""_

Later gives error on iphone- Airplay - Unable to connect to <name of device>.

TIA

Rajat

**Comment by GuillaumeRossolini** [comment]
Hi,

Perhaps it is a power management issue? Try this:
```
iw wlan0 get power_save && iwconfig 2>/dev/null | grep "Power Management"*
```
If this says "Power save: on", you'll need to change `/etc/rc.local` with this new line (on its own) before the exit
```
/sbin/iwconfig wlan0 power off
```
